frontend/package.json

```diff
@@ -38,6 +38,8 @@
     "@types/enzyme": "^3.10.3",
     "@zeit/next-css": "^1.0.1",
     "autobahn": "19.7.3",
+    "big.js": "^5.2.2",
```

Comment:
We don't neet it. We use `bn.js` instead.

Reply:
deleted